cpp/ql/src/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -219,10 +240,41 @@ class DefinitionByReferenceNode extends Node {
   }
 }
 
+/**
+ * A `Node` corresponding to a variable in the program, as opposed to the
+ * value of that variable at some particular point. This can be used for
+ * modelling flow in and out of global variables.
```

Comment:
Is there value or cost to the fact that we also model non-global variables here?

Reply:
Value: yes. Callers might use it to crudely overapproximate flow through fields, or they might at least use it for flow through `static` fields. It's only for the sake of bug-for-bug backwards compatibility that I haven't added flow through static fields in `DefaultTaintTracking`.

Cost: not much. It adds a small number of tuples to the IPA type -- the number of variables in the program, which should be much less than the number of instructions.